### ISSUE TITLE
How to Train FCN8s with CamVid Dataset

### ISSUE THREAD

**Issue by liqisa** [meetshah1995/pytorch-semseg#201]
Hi,Can you release a config.yaml about training fcn8s in camvid? I think i can't set  proper parameters such as lr batch_size ,iterations and so on... Thanks!

**Comment by soans1994** [comment]
> Hi,Can you release a config.yaml about training fcn8s in camvid? I think i can't set proper parameters such as lr batch_size ,iterations and so on... Thanks!

how did you get train test split? i have raw images and label images
